build.gradle

```diff
@@ -121,6 +122,10 @@ OpenWhiskPlatform.zipFileName =
     System.env['zip_file_name'] ?:
         (rootProject.findProperty('zipFileName') ?: 'OpenWhisk_CLI')
 
+OpenWhiskPlatform.packageVersion =
+        System.env['packageVersion'] ?:
+                (rootProject.findProperty('packageVersion') ?: 'latest')
```

Comment:
Per my comment above, this can become `project.ext.packageVersion =`.  Also, I’m not sure I’d use an environment variable because it hides the “I’m building a release now” from the command line.  If we have a variable set in a Travis build, we should pass it on the command line as the build parameter (as you provide here).

Reply:
Done. I remove the configuration as a system env.